cli/crates/server/src/servers.rs

```diff
@@ -265,29 +274,40 @@ async fn run_schema_parser() -> Result<(), ServerError> {
         .join(SCHEMA_PARSER_DIR)
         .join(SCHEMA_PARSER_INDEX);
 
-    let output = Command::new("node")
-        .args(&[
-            &parser_path.to_str().ok_or(ServerError::CachePath)?,
-            &environment
-                .project_grafbase_schema_path
-                .to_str()
-                .ok_or(ServerError::ProjectPath)?,
-        ])
-        .current_dir(&environment.project_dot_grafbase_path)
-        .stderr(Stdio::piped())
-        .spawn()
-        .map_err(ServerError::SchemaParserError)?
-        .wait_with_output()
-        .await
-        .map_err(ServerError::SchemaParserError)?;
+    let environment_variables: std::collections::HashMap<_, _> = environment_variables().collect();
+
+    let output = {
+        let mut node_command = Command::new("node")
+            .args(&[
+                &parser_path.to_str().ok_or(ServerError::CachePath)?,
+                &environment
+                    .project_grafbase_schema_path
+                    .to_str()
+                    .ok_or(ServerError::ProjectPath)?,
+            ])
+            .current_dir(&environment.project_dot_grafbase_path)
+            .stderr(Stdio::piped())
+            .spawn()
+            .map_err(ServerError::SchemaParserError)?;
+
+        let node_command_stdin = node_command.stdin.as_mut().unwrap();
```

Comment:
Is there a valid case this could happen? I'd use `expect` if it shouldn't happen or handle the error if it should

Reply:
Good point, changed to `expect()`.